package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chamaeleonidae/chmln",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Are we okay with releasing these changes as a `patch`?

We previously made sure that we exported `init/identify/track` like valid functions even if `window.chmln` wasn't ready yet, is there any chance that with this new code, they might run into `is not a function` and this becomes a breaking change?

Reply:
I'm ok with `1.0.1` if we are 100% sure that this is a backward-compatible change, otherwise, it must be a major bump to `2.0.0` to avoid unwanted side effects to users that have installed it in their apps accepting a larger version range. 

(e.g `~1.0.0` or `^1.0.0`)

Reply:
@nbfontana from my testing this is indeed a backwards compatible. I was able to import specific methods of `init` `identify` and `track` (which were available in past versions) into our dashboard project and reference them successfully.